service-food/src/main/kotlin/de/hpi/cloud/food/crawler/openmensa/canteens/Ulf.kt

```diff
@@ -0,0 +1,48 @@
+package de.hpi.cloud.food.crawler.openmensa.canteens
+
+import de.hpi.cloud.common.entity.Id
+import de.hpi.cloud.food.crawler.ParsedMensaMeal
+import de.hpi.cloud.food.crawler.openmensa.CanteenData
+import de.hpi.cloud.food.fixtures.labelFixture
+
+object Ulf : CanteenData(Id("ulfsCafe"), 112) {
+    val PRICE_REGEX = Regex("^(.*?)\\s+(\\d,\\d\\d?)\\s?€?\$")
```

Comment:
```suggestion
    val PRICE_REGEX = Regex("^(.*?)\\s+(\\d,\\d{1,2})\\s?€?\$")
```

Reply:
nah, this should not matter

Reply:
It's just personal preference, but not important right now